.github/workflows/coverage.yml

```diff
@@ -91,8 +91,19 @@ jobs:
             go.mod
             go.sum
       - name: install
+        run: sudo make install
+        if: env.GIT_DIFF != ''
+      - name: build
         run: GOOS=linux GOARCH=${{ matrix.goarch }} make build
-        if: "env.GIT_DIFF != ''"
+        if: matrix.goarch == 'amd64' && env.GIT_DIFF != ''
+      - name: build
+        run: |
+          sudo apt-get install gcc-arm-linux-gnueabihf
```

Comment:
what do you think groupping deps in one line `sudo apt-get install gcc-arm-linux-gnueabihf g++-8-arm-linux-gnueabihf`?

Reply:
Cool, would make those changes